Tools/WinML-Dashboard/public/requirements.txt

```diff
@@ -0,0 +1,17 @@
+--extra-index-url https://tiagoshibata.github.io/PyPI/simple
```

Comment:
Could you replace this with a microsoft page rather than a personal one.

Reply:
Yes, you'll have to fork [tiagoshibata/PyPI](https://github.com/tiagoshibata/PyPI) to a Microsoft org and you should be all set

Reply:
The PyPI repo and web interface are created with [dumb-pypi](https://github.com/chriskuehl/dumb-pypi)